src/ext/BalExtension/wixstdba/WixStandardBootstrapperApplication.cpp

```diff
@@ -576,6 +566,7 @@ class CWixStandardBootstrapperApplication : public CBalBaseBootstrapperApplicati
         __in HRESULT /*hrStatus*/
         )
     {
+        UpdateCacheProgress(100); // Make sure cache progress got set to 100%.
```

Comment:
What if caching failed? Should we really mark it 100%? Why isn't it always 100% before we get here on success?